TYPO3.Neos/Documentation/CreatingASite/TypoScript/InsideTypoScript.rst

```diff
@@ -392,6 +392,53 @@ To sum it up: When implementing a TypoScript object, it should not access its co
 directly, but instead use a property. In the TypoScript object's prototype, a default mapping
 between a context variable and the prototype can be set up.
 
+Default Context Variables
+=========================
+
+Neos exposes some default variables to the TypoScript context that can be used to control page rendering
+in a more granular way.
+
+* ``node`` can be used to get access to the current node in the node tree and read its properties, such as::
+
+    # Make the node available in the template
+    node = ${node}
+
+    # Get access to node data directly (careful: this relies on the internal node model and thus can change!)
+    nodePath = ${node.nodeData.path}
```

Comment:
Woah, please don't mention `nodeData` at all. It is not public API and a warning will not keep people from using it… If needed rather show `property('__path')` (is that right?).